onnxruntime/core/framework/prepacked_weights_container.h

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <unordered_map>
+#include <unordered_set>
+#include <string>
+#include <cstdint>
+
+#include "core/framework/buffer_deleter.h"
+#include "core/framework/tensor_shape.h"
+#include "core/framework/allocator.h"
+#include "core/platform/ort_mutex.h"
+#include "core/framework/murmurhash3.h"
+
+namespace onnxruntime {
+
+struct PrepackedWeight final {
+  // Some weights may be associated with multiple pre-packed buffers.
```

Comment:
nit: if this is storing multiple weights, PrepackedWeights would be a clearer name.

Reply:
Sure fixed